packages/crayons-core/src/components/form-control/form-control.tsx

```diff
@@ -457,6 +458,53 @@ export class FormControl {
           );
         }
         break;
+      case 'DEPENDENT_FIELD':
```

Comment:
for ` case 'DROPDOWN':` the below lines are already present. Can you just update the the component rendered for this DEPENDENT_FIELD type and reuse that code ?

Reply:
Done.

Reply:
I dont see the changes . Can you check ?. I still see a separate case statement for `DEPENDENT_FIELD` type @prasanna-vijayan